### ISSUE TITLE
Builds not triggering with 1.6.1 and Bitbucket Server 7.0.1

### ISSUE THREAD

**Issue by s1rlon** [jenkinsci/bitbucket-push-and-pull-request-plugin#27]
After upgrading to 1.6.1 version of the plugin jobs fail to trigger, downgrading to 1.6.0 fixes the issue.
Jenkins ver. 2.176

From the Jenkins logs:
```
May 10, 2019 9:43:47 AM io.jenkins.plugins.bitbucketpushandpullrequest.BitBucketPPRHookReceiver doIndex
INFO: Response sent.
May 10, 2019 9:43:47 AM io.jenkins.plugins.bitbucketpushandpullrequest.environment.BitBucketPPRAdditionalParameterServerRepositoryEnvironmentContributor buildEnvironmentFor
INFO: Injecting env vars because of server push cause.
May 10, 2019 9:43:47 AM io.jenkins.plugins.bitbucketpushandpullrequest.environment.BitBucketPPRAdditionalParameterServerRepositoryEnvironmentContributor buildEnvironmentFor
WARNING: Problem injecting env variables: Cause = null
May 10, 2019 9:43:47 AM io.jenkins.plugins.bitbucketpushandpullrequest.environment.BitBucketPPRAdditionalParameterRepositoryEnvironmentContributor buildEnvironmentFor
INFO: Injecting env vars because of push cause.
May 10, 2019 9:43:47 AM io.jenkins.plugins.bitbucketpushandpullrequest.BitBucketPPRPollingRunnable run
SEVERE: Failed to record SCM polling
java.lang.ClassCastException: io.jenkins.plugins.bitbucketpushandpullrequest.action.BitBucketPPRServerRepositoryAction cannot be cast to io.jenkins.plugins.bitbucketpushandpullrequest.action.BitBucketPPRRepositoryAction
        at io.jenkins.plugins.bitbucketpushandpullrequest.cause.repository.BitBucketPPRRepositoryCause.getRepositoryPayLoad(BitBucketPPRRepositoryCause.java:41)
        at io.jenkins.plugins.bitbucketpushandpullrequest.environment.BitBucketPPRAdditionalParameterRepositoryEnvironmentContributor.buildEnvironmentFor(BitBucketPPRAdditionalParameterRepositoryEnvironmentContributor.java:41)
        at hudson.model.Run.getEnvironment(Run.java:2379)
        at hudson.model.AbstractBuild.getEnvironment(AbstractBuild.java:864)
        at hudson.plugins.git.GitSCM.getParamExpandedRepos(GitSCM.java:484)
        at hudson.plugins.git.GitSCM.compareRemoteRevisionWithImpl(GitSCM.java:693)
        at hudson.plugins.git.GitSCM.compareRemoteRevisionWith(GitSCM.java:658)
        at hudson.scm.SCM.compareRemoteRevisionWith(SCM.java:400)
        at hudson.scm.SCM.poll(SCM.java:417)
        at hudson.model.AbstractProject._poll(AbstractProject.java:1390)
        at hudson.model.AbstractProject.poll(AbstractProject.java:1293)
        at io.jenkins.plugins.bitbucketpushandpullrequest.BitBucketPPRPollingRunnable.run(BitBucketPPRPollingRunnable.java:66)
        at hudson.util.SequentialExecutionQueue$QueueEntry.run(SequentialExecutionQueue.java:119)
        at jenkins.util.ContextResettingExecutorService$1.run(ContextResettingExecutorService.java:28)
        at jenkins.security.ImpersonatingExecutorService$1.run(ImpersonatingExecutorService.java:59)
        at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511)
        at java.util.concurrent.FutureTask.run(FutureTask.java:266)
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
        at java.lang.Thread.run(Thread.java:748)

May 10, 2019 9:43:47 AM io.jenkins.plugins.bitbucketpushandpullrequest.BitBucketPPRTrigger$1 onPollError
INFO: Called onPollError
```

**Comment by cdelmonte-zg** [comment]
@s1rlon Bug fixed just now in the version 1.6.2
Do you confirm?
Thanks

**Comment by s1rlon** [comment]
With version 1.6.2 builds are being triggered again
